go/tasks/pluginmachinery/core/phase.go

```diff
@@ -159,6 +159,12 @@ func PhaseInfoWaitingForResources(t time.Time, version uint32, reason string) Ph
 	return pi
 }
 
+func PhaseInfoWaitingForResourcesInfo(t time.Time, version uint32, reason string, info *TaskInfo) PhaseInfo {
```

Comment:
Can you add `// Deprecated: Please use PhaseInfoWaitingForResourcesInfo instead.` to the `PhaseInfoWaitingForResources` func?

Reply:
Done